CONTRIBUTING.md

```diff
@@ -1,52 +1,72 @@
 # Contributing to OpenEBS
 
-Great!! We are always on the lookout for more OpenEBS hackers. You can get started by reading this [Overview](./contribute/design/README.md)
+Great!! We are always on the lookout for more OpenEBS hackers. You can get started by reading this [overview](./contribute/design/README.md)
 
-OpenEBS is innovation in OpenSource. You are welcome to contribute in any way you can and all the help you can provide is very much appreciated. 
+Firstly, if you are unsure or afraid of anything, just ask or submit the issue or pull request anyways. You won't be yelled at for giving your best effort. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contributions and don't want a wall of rules to get in the way of that.
```

Comment:
submit an issue
won't = will not
you'll = you will 
don't = do not
get in the way of = hamper

Reply:
This paragraph has been directly taken out of https://github.com/hashicorp/packer/blob/master/.github/CONTRIBUTING.md
I like it very much and that's why I put it here too. I do agree with your review. But the context here is to make the person feel comfortable. So we could let the grammatical slide a bit here.